### PR TITLE
Add REST methods for stage channels

### PR DESCRIPTION
**Pull request by bsian03**
~~Caveat page isn't live yet~~

### CODE REVIEW

File: lib/structures/Guild.js [hunk 1]
```diff
@@ -601,6 +601,19 @@ class Guild extends Base {
         return this._client.editGuildTemplate.call(this._client, this.id, code, options);
     }
 
+    /**
+    * Update a user's voice state - See [caveats](https://discord.com/developers/docs/resources/guild#update-others-voice-state-caveats)
+    * @param {Object} options The properties to edit
+    * @param {String} options.channelID The ID of the channel the user is currently in
+    * @param {Date?} [options.requestToSpeakTimestamp] Sets the user's request to speak - this can only be used when the `userID` param is "@me"
+    * @param {Boolean} [options.suppress] Toggles the user's suppress state
+    * @param {String} userID The user ID of the user to update
```
Comment (bsian03) [inline lib/structures/Guild.js]:
```suggestion
    * @param {String} [userID="@me"] The user ID of the user to update
```

File: lib/Client.js [hunk 2]
```diff
@@ -1122,6 +1122,24 @@ class Client extends EventEmitter {
...
     }
 
+    /**
+    * Update a user's voice state - See [caveats](https://discord.com/developers/docs/resources/guild#update-others-voice-state-caveats)
+    * @param {String} guildID The ID of the guild
+    * @param {Object} options The properties to edit
+    * @param {String} options.channelID The ID of the channel the user is currently in
+    * @param {Date?} [options.requestToSpeakTimestamp] Sets the user's request to speak - this can only be used when the `userID` param is "@me"
+    * @param {Boolean} [options.suppress] Toggles the user's suppress state
+    * @param {String} userID The user ID of the user to update
```
Comment (bsian03) [inline lib/Client.js]:
```suggestion
    * @param {String} [userID="@me"] The user ID of the user to update
```

**Review by bsian03 — COMMENTED** [review]
On mobile, don't mind me

File: lib/Client.js [hunk 3]
```diff
@@ -1122,6 +1122,24 @@ class Client extends EventEmitter {
...
+    * Update a user's voice state - See [caveats](https://discord.com/developers/docs/resources/guild#update-others-voice-state-caveats)
+    * @param {String} guildID The ID of the guild
+    * @param {Object} options The properties to edit
+    * @param {String} options.channelID The ID of the channel the user is currently in
+    * @param {Date?} [options.requestToSpeakTimestamp] Sets the user's request to speak - this can only be used when the `userID` param is "@me"
+    * @param {Boolean} [options.suppress] Toggles the user's suppress state
+    * @param {String} [userID="@me"] The user ID of the user to update
+    * @returns {Promise}
+    */
+    editGuildVoiceState(guildID, options, userID = "@me") {
```
Comment (HeadTriXz) [inline lib/Client.js:1135]: Consider using (guildID, memberID, options) instead.

Comment (bsian03) [reply lib/Client.js:1135]: Options is required. Required parameter following optional parameter is also frowned upon.

Comment (HeadTriXz) [reply lib/Client.js:1135]: Fair enough.

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**